src/OrderLinesList/OrderLinesList.js

```diff
@@ -25,8 +25,11 @@ import {
   FUNDS,
 } from '../components/Utils/resources';
 import OrderLinesFilters from './OrderLinesFilters';
-import { filterConfig } from './OrdersLinesFilterConfig';
 import Details from './Details';
+import {
+  filterConfig,
+  searchableIndexes,
```

Comment:
you you please move `searchableIndexes` to another file?  e.g. `OrdersLinesSearchConfig`